### ISSUE TITLE
Small index bug in street_readParams()

### ISSUE THREAD

**Issue by jlhomme** [USEPA/Stormwater-Management-Model#112]
On lines 109-110 of street.c, in `street_readParams()`
there is a small index bug that could return an error when the input are actually correct (`k` is equal to 4 when it comes to this block).

```
    // --- read street backing parameters
    if (ntoks > 8)
    {
        if (!getDouble(tok[8], &x[8]) || x[k] < 0.0)                   //<<<< incorrect index k
            return error_setInpError(ERR_NUMBER, tok[k]);              //<<<< incorrect index k
        if (x[8] > 0.0)
        {
            if (ntoks < 11) return error_setInpError(ERR_ITEMS, "");
            for (k = 9; k <= 10; k++)
                if (!getDouble(tok[k], &x[k]) || x[k] <= 0.0)
                    return error_setInpError(ERR_NUMBER, tok[k]);
        }
    }
```
This should be
```
    // --- read street backing parameters
    if (ntoks > 8)
    {
        if (!getDouble(tok[8], &x[8]) || x[8] < 0.0)			// EPA bug, x[8] not x[k]
            return error_setInpError(ERR_NUMBER, tok[8]);
        if (x[8] > 0.0)
        {
            if (ntoks < 11) return error_setInpError(ERR_ITEMS, "");
            for (k = 9; k <= 10; k++)
                if (!getDouble(tok[k], &x[k]) || x[k] <= 0.0)
                    return error_setInpError(ERR_NUMBER, tok[k]);
        }
    }
```

**Comment by jlhomme** [comment]
Hi @michaeltryby I just tried to publish a branch with the fix so I could do a PR, but Github says I don't have write access to the repo so won't let me do that.
Are you happy to give me write access?

**Comment by LRossman** [comment]
@jlhomme I took it upon myself to commit your fixes. Please see the 5-2-3_bug_fixes branch and its PR #134.

**Comment by cbuahin** [comment]
@jlhomme, I do not think we have the infrastructure to provide everyone the permissions to create branches on the EPA repository. I don't think that we are able to realistically manage such a system. A more workable system that we are considering might be allowing people to fork the EPA SWMM repo to their personal repo, where they can make pull requests to the EPA repo. More critically, we are discussing a code of conduct, a document clarifying that contributions will be in the public domain, and a way to give credit to contributors who discover or fix bugs so that we are able to encourage the excellent contributions from folks such as yourself. I apologize that there is not a definitive path but hopefully we will get there sooner rather than later. @LRossman, as a start I think it will be good to note the contributors who discover bugs and provide suggestions in PRs.

**Comment by jlhomme** [comment]
@cbuahin yes that sounds sensible.

**Comment by cbuahin** [comment]
Addressed in #134 .
